### DISCUSSION TITLE
Not able to delete fields under service category

### DISCUSSION THREAD

**Discussion by CA-NikhilKumar** [madrecha/jamku#1156] [QA]
### Feature Request

I am trying to delete some field lable created earlier. But it shows an error that it is linked to 129 tasks
![image](https://user-images.githubusercontent.com/120717957/208074852-b7a1df03-70c5-4a7f-8be8-dc180ba22ba7.png)

**Comment by adarshmadrecha** [comment]
In this case you have 2 options
1. Delete the field in those tasks. It can be done via Excel Import. Reach out to Darshana for any help in this regards.
2. Deactivate the field.

**Reply by CA-NikhilKumar** [reply]
Is there is no option to delete? Yesterday Yogesh mentioned it can be done
